services/s3/src/main/java/software/amazon/awssdk/services/s3/AwsS3V4Signer.java

```diff
@@ -20,7 +20,7 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import software.amazon.awssdk.auth.signer.AbstractAws4Signer;
+import software.amazon.awssdk.auth.signer.internal.AbstractAws4Signer;
```

Comment:
Since this crosses module boundaries we'll need to move it out of internal and annotate with Protected.

Reply:
done